test/hsts.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const assert = require('assert');
+const fetch = require('node-fetch');
+
+const HOSTNAMES = [
+  // not an exhaustive list, just enough to catch accidental removal
+  'whatwg.org',
+  'blog.whatwg.org',
+  'dom.spec.whatwg.org',
+  'participate.whatwg.org',
+  'resources.whatwg.org',
+  'spec.whatwg.org',
+  'wiki.whatwg.org',
+];
+
+describe('strict-transport-security header', function() {
+  for (const hostname of HOSTNAMES) {
+    specify(hostname, async function() {
+      // redirecting is a failure since we might then test the wrong server
+      const response = await fetch(`https://${hostname}/`, { redirect: 'manual' });
+      assert.strictEqual(response.status, 200);
+      let value = response.headers.get('strict-transport-security');
+      // Trim any trailing semicolon.
+      if (value.endsWith(';')) {
```

Comment:
Ideally we're consistent on this.

Reply:
Yes, but I didn’t want to change that at the same time.